src/prelogin-payload.ts

```diff
@@ -180,6 +186,19 @@ class PreloginPayload {
     };
   }
 
+  createTraceIdOption() {
+    const buffer = new WritableTrackingBuffer(36);
+    // generate a random series of bytes to use as the TraceID
+    // used for debugging purposes
+    for (let i = 0; i < 36; i++) {
+      buffer.writeUInt8(randomBytes(1)[0]);
```

Comment:
for now the traceID is just random. We could consider actually making it based on PID or at least persistent through the session, but I think it's fine if it's always just a unique ID.

Also I'm going to change how to we generate the bytes to be less silly